### ISSUE TITLE
make blog section as mobile responsive

### ISSUE THREAD

**Issue by sabesansathananthan** [sabesansathananthan/sabesansathananthan.github.io#3]
in index.html file blog section is not align with mobile phone and tablets want to fix
![Image from iOS](https://user-images.githubusercontent.com/36799589/67458682-7610a980-f654-11e9-91f7-c7fd329c2615.png)

**Comment by sabesansathananthan** [comment]
@timd3v  could you please suggest any ideas ???

**Comment by timd3v** [comment]
Yes, by the looks of it, I would suggest switching the flex direction to column at smaller screen sizes.

Before flex direction:
![Capture](https://user-images.githubusercontent.com/41599429/67474698-ce43ad80-f64c-11e9-8bd3-37ace36d6467.PNG)

After flex direction:
![Capture2](https://user-images.githubusercontent.com/41599429/67474709-d26fcb00-f64c-11e9-8b40-c0404c8f19d0.PNG)

**Comment by sabesansathananthan** [comment]
@timd3v  participate in this hackathon. 
https://hacktoberfest.digitalocean.com/details#quality-standards
